hardhat.config.js

```diff
@@ -14,6 +14,11 @@ module.exports = {
                 enabled: true,
                 runs: 1000,
             },
+            outputSelection: {
+                "*": {
+                    "*": ["storageLayout"],
+                },
+            },
```

Comment:
what does this do?

Reply:
Smockit needs access to the internal storage layout to help it mock calls, and this enables that. See: https://github.com/ethereum-optimism/optimism/tree/develop/packages/smock

Reply:
Also I was wrong on the above, its only needed for `smoddit` which we don't use yet. Have removed this config for now